### ISSUE TITLE
Remove rules with empty attributes

### ISSUE THREAD

**Issue by wmertens** [thysultan/stylis.js#58]
See https://github.com/styled-components/styled-components/issues/1110

It would be great if stylis removed all these empty examples:

```
.emptyAttr { color: ; } // should be removed
.spuriousSemi { color: black; ; ; ; ; } // should only keep color: black
.emptySelector { } // should be removed
```

**Comment by thysultan** [comment]
The last two are removed.

```css
.spuriousSemi { color: black; ; ; ; ; } // should only keep color: black
.emptySelector { } // should be removed
```

**Comment by thysultan** [comment]
RE: The first one. Is there a reason you want this behavior? It is possible but i don't think we should handle it.

**Comment by wmertens** [comment]
It follows from the styled-components API. If you have a component like

```js
const Box = styled.div`
    flex-direction: ${p => p.column && 'column'};
`
```

and you don't set `column`, you end up with an empty `flex-direction` rule as well as all the prefixed versions of it.

I know the alternative is to write

```js
const Box = styled.div`
    flex-direction: ${p => p.column ? 'column' : 'row'};
`
```

but then I'm emitting lots of CSS that has no effect since `row` is the default, and it also makes it impossible to only override some rules if required by the props.

So in the interests of cleaner CSS, less work for the browser and a smaller SSR payload, it would be nice if stylis removed empty rules.

**Comment by thysultan** [comment]
Included in the 3.2.16 patch.
